tests/adapters/base.py

```diff
@@ -30,55 +70,167 @@ def test_initialize_model_input(self, test_adapter):
         """
         Test that initalizing model input does as intended.
         """
+        set_values = [0.9, 1.1]
+        test_adapter._output.get_writable_timeseries_view(
+            ("Emissions", "CO2"),
+            ("World",),
+            "GtC/yr",
+            [-10, 0, 10],
+            ParameterType.POINT_TIMESERIES,
+        ).set(set_values)
+
         assert not test_adapter._initialized
-        # TODO test for missing but mandatory parameter
-        test_adapter.initialize_model_input()
+        # start and end time should be set here as they're part of setting the
+        # timeseries views of emissions/concentrations/other drivers
+        test_adapter.initialize_model_input(start_time, end_time)
         assert test_adapter._initialized
+        # TODO test for start time and end time being passed properly
```

Comment:
Hmm, don't know how we should test if they are used correctly

Reply:
I think this would have to be model specific (and so maybe this test method would have to be abstract) but e.g. with MAGICC, you'd test that the start and end times actually get passed into the cfg file which defines how the run is to be done

Reply:
let's have a play when we actually have some adapters to work with